optimade/client/__init__.py

```diff
@@ -0,0 +1,21 @@
+"""This module implements an OPTIMADE client that can be called
+from Python code with
+
+```python
+from optimade.client import OptimadeClient
+client = OptimadeClient()
+client.get('elements HAS "Ag")
+```
+
+or from the command-line with
+```shell
+optimade-get 'elements HAS "Ag"'
```

Comment:
Could you explain a bit more about how I should use this from the command line ? I did not manage to get it working on my laptop.